src/test/resources/htsjdk/samtools/BAMFileIndexTest/htsget_config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Should this live with the script?  Can we add a README explaining this somewhere?

Reply:
Moved to a folder with the script and added a readme